src/settings/project/mod.rs

```diff
@@ -37,17 +37,8 @@ impl Project {
         project_type: ProjectType,
         init: bool,
     ) -> Result<Project, failure::Error> {
-        let project = Project {
-            name: name.clone(),
-            project_type: project_type.clone(),
-            private: Some(false),
```

Comment:
I guess the default of Option<bool> is None?

Reply:
mmm yes it is. I should likely update to use a more tailored default implementation than the one that can be derived. or make `new` define it like it does with project_type.